Kennie-Simz/Project-Uno
# #167105841 User can view all property

#### What does this PR do?
- It allows the user to view all property.

#### Description of Task to be completed?
- The user needs to be able to view all property on the website or app.

#### How should this be manually tested?
- Clone this repo
```
$ git clone https://github.com/Kmozart/Project-Uno
```
- Checkout to this branch
```
$ git checkout ch-user-can-view-all-property-167105841
```
- Install dependencies
```
npm install
```
- Run the app
```
$ npm run start
```
- Check out the route below on Postman
```
localhost:4000/api/v1/auth/v1/property
```
